org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -8,14 +8,13 @@
 var request = require('request-promise');
 var OpenT2T = require('opent2t').OpenT2T;
 var Crypto = require('crypto');
-var accessTokenInfo = require('./common').accessTokenInfo;
 
 /**
 * This translator class implements the "Hub" interface.
 */
 class Translator {
-    constructor(accessToken) {
-        this._accessToken = accessToken;
+    constructor(authTokens) {
+        this._authTokens = authTokens;
```

Comment:
CLI etc just pass this object directly from the onboarder to the hub translator without looking at it.  Hub controller tests are the only ones that I can find that look at the token directly and that's fixed here: [https://github.com/openT2T/cloud/pull/20](https://github.com/openT2T/cloud/pull/20)

---
In reply to: [101099112](https://github.com/openT2T/translators/pull/160#discussion_r101099112) [](ancestors = 101099112)